Runtime/Core/BucketManagement/BucketStore.cs

```diff
@@ -117,6 +117,8 @@ public CreateBucketResponse CreateBucket(CreateBucketRequest request)
                 {
                     BucketName = loggingBucketName,
                     BucketRegionName = request.Region,
+                    CannedACL = S3CannedACL.LogDeliveryWrite,
+                    ObjectOwnership = ObjectOwnership.BucketOwnerPreferred
```

Comment:
This is not the correct fix. I believe we had discussed fixing this by not using ACLs entirely since it's not longer allowed to use ACLs as a best practise. We should instead solve this using S3 access policies. That needs to happen and is a blocker for launch.

Reply:
Sure thing, sorry for not providing context. We currently use ACLs to allow S3 to add access logs around bucket usage. Recently S3 changed their default bucket rules which meant that our ACL would not work by default, and customers wouldn't be able to spin up their buckets. The fix here is the right fix for an ACL based approach, but we have been informed that we shouldn't be using ACLs to begin with.

The alternative is to use a bucket policy which is the AWS standard. It is outlined here: https://docs.aws.amazon.com/AmazonS3/latest/userguide/enable-server-access-logging.html

This fix itself can happen as a separate PR, but needs to happen.